src/index.ts

```diff
@@ -219,4 +226,9 @@ export {
 	profiler,
 	registerCustomXPathFunction,
 	registerXQueryModule,
+	compileXPathToJavaScript,
```

Comment:
Can you alfabetize this?

Reply:
Can you also export this on line 129? That's the place where we export fontoxpath for use directly in the browser with globals. The old-fashioned way.